.gitmodules

```diff
@@ -6,7 +6,7 @@
 	url = https://github.com/cypherstack/flutter_liblelantus.git
 [submodule "crypto_plugins/monero_c"]
 	path = crypto_plugins/monero_c
-	url = https://git.mrcyjanek.net/mrcyjanek/monero_c
+	url = git@github.com:cypherstack/monero_c
```

Comment:
Hey! The upstream source code is also available at https://github.com/mrcyjanek/monero_c